scripts/batch.py

```diff
@@ -0,0 +1,251 @@
+"""This script processes SIC code batch data through Survey Assist API.
+It is based on configurations specified in a .toml file.
+Prior to invocation, ensure to run the following CLI commands:
+> gcloud config set project "valid-priject-name"
+> gcloud auth application-default login
+By defalut the output file is appended to, not overwritten.
+Delete it before running, or change the name if this is not want you want.
+
+Run from the root of the project as follows:
+
+path/to/python /home/user/survey-assist-utils/scripts/batch.py
```

Comment:
if you are in the root directory you don't need anything prior to scripts.
Also, why is there path/to/python? I assumed that there will be something like `poetry run path/to/script` or `python path/to/script`?

Reply:
You have specified it yourself on line 31: poetry run python scripts/batch.py